kuma/wiki/admin.py

```diff
@@ -260,12 +260,26 @@ def rendering_info(obj):
 
 
 def akismet_data_as_dl(akismet_data):
+    """Format Akismet data as a definition list."""
+    favorites = (
```

Comment:
Thanks for the example code. The sequence still seems like a better method to me, because it communicates the order with the least code. Encoding it as a dictionary opens questions like "do more items get added later?" and "what does it mean if the value is a non-integer?".